bifromq-apiserver/src/main/java/com/baidu/bifromq/apiserver/http/handler/HTTPSubHandler.java

```diff
@@ -88,6 +95,7 @@ public CompletableFuture<FullHttpResponse> handle(@Parameter(hidden = true) long
             String topicFilter = getHeader(HEADER_TOPIC_FILTER, req, true);
             QoS subQoS = getRequiredSubQoS(req);
             String inboxId = getHeader(HEADER_INBOX_ID, req, true);
+            String deliverKey = getHeader(HEADER_DELIVER_KEY, req, false);
```

Comment:
add a check to verify following condition: when subBrokerId is not 0 or 1, the delivererKey is mandatory